frontend/src/common/backend.ts

```diff
@@ -257,7 +257,7 @@ class VaultService {
   }
 
   public async accessToken(vaultId: string): Promise<string> {
-    return axiosAuth.get(`/vaults/${vaultId}/access-token`, { headers: { 'Content-Type': 'text/plain' } })
+    return axiosAuth.get(`/vaults/${vaultId}/access-token?ignoreArchivedFlag=true`, { headers: { 'Content-Type': 'text/plain' } })
```

Comment:
Should this be hard-coded? Even if the frontend always needs to ignore this flag, I'd prefer to expose the parameter via the method signature (`accessToken(vaultId: string, evenIfArchived = false): Promise<string>`) and explicitly set this boolean to true from the component when needed.

Reply:
Applied in [ac513a6](https://github.com/cryptomator/hub/pull/230/commits/ac513a677e950ad490821a75a823408851332598)